### DISCUSSION TITLE
Authentication

### DISCUSSION THREAD

**Discussion by PhilGsk** [dasniko/testcontainers-keycloak#35] [General]
I want to integrate the keycloak testcontainer in my integration tests. Unfortunately when i try to create a role or delete an existing user i get a HTTP 403. Is it possible to test a mockMvc request with granted user?

**Comment by dasniko** [comment]
With this Keycloak Testcontainer you'll get a regular Keycloak server for your tests. No need to mock anything. That's the general idea behind the Testcontainers project.
